on_the_porch/postgres_stuff/transfer.py

```
@@ -0,0 +1,100 @@
+                        continue
+
+
+if __name__ == "__main__":
+    # Run filtering by default
+    filter_911_csvs_for_dorchester()
+
+    # Optional: set UPLOAD_TO_DB=1 to upload CSVs to Postgres after filtering
+    if os.environ.get("UPLOAD_TO_DB") == "1":
+        # --- CONFIGURATION ---
+        PG_USER = "user1"
+        PG_PASSWORD = "BbWTihWnsBHglVpeKK8XfQgEPDOcokZZ"
```

Ruff (0.14.1)

Possible hardcoded password assigned to: "PG_PASSWORD"

(S105)